Automattic/wp-calypso
# Gravatar: Fix resend email not working for new user & Add countdown time for the resending email function

Related to:

* 105699-Automattic/gravatar
* 105691-Automattic/gravatar

## Proposed Changes

* Enable `createAccount` for the resending email function
* Add countdown time for the resending email function
* Remove duplicated state selector (`[get-magic-login-is-fetching-email.js](https://github.com/Automattic/wp-calypso/pull/81544/files#diff-50db5d70f7263f2beaa955104b06e4e2c09dbd1a9da16f52b9214f0144c0398e)`), which was created from [this PR](https://github.com/Automattic/wp-calypso/pull/81258/files#diff-50db5d70f7263f2beaa955104b06e4e2c09dbd1a9da16f52b9214f0144c0398e)

## Testing Instructions



* Setup Calypso by following [this doc](https://github.com/Automattic/wp-calypso#getting-started), and check out this PR
* Log out your account
* Run `yarn start`, and create a new account via [this URL](http://calypso.localhost:3000/log-in/link?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D434cd756529bcf3f73f447f1a358128078b4180858d4b8d59bf82cf108d799e1%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1)
* You will see the resending email countdown:
<img width="575" alt="截圖 2023-09-08 下午3 37 09" src="https://github.com/Automattic/wp-calypso/assets/21308003/7e2381b3-77ac-47b7-b93b-cc82d592b930">

* After the countdown finishes, click the resend email function, you will receive the signup email
* Log out of your account, and try to log in to your account, the resend email function should still work the same

## Pre-merge Checklist
